backend/src/main/resources/db-migrations/common/v0-[common]-schema-init.sql

```diff
@@ -59,11 +59,30 @@ CREATE INDEX `ix_alert_type_code` on `alert_type` (`code`);
 
 
 
+CREATE TABLE `preferred_language` (
```

Comment:
As mentioned in previous comment, I think this should just be called `language`. The reason is that the data within this table could be used for more than just user preferences.

Reply:
updated